shared/census_mapping.json

```diff
@@ -57,7 +57,7 @@
       "human_readable_name": "Asian"
     },
     "census_acs_dp_05.hc03_vc98": {
-      "joined_column_name": "native_hawaiian_and_other_pacific_islander_not_hispanic_or_latino",
+      "joined_column_name": "hawaiian_and_pacific_islander_not_hispanic_or_latino",
```

Comment:
how is this causing the issue now? is there a max column length? can we add a test somewhere to make sure we're not over if that's the case?

Reply:
Limit is 63 characters, I do not think this is worth adding tests for though

Reply:
can we add an assert so at least we get an error?

Reply:
Done